app/views/provisioning_templates/_alerts.html.erb

```diff
@@ -1,5 +1,6 @@
 <% if @template.locked? -%>
   <%= locked_warning(::Template.find(params[:id])) %>
 <% else -%>
+  <%= infobox_safemode if Setting[:safemode_render] %>
   <%= infobox_functions_and_macros("4.4.3ProvisioningTemplates") %>
```

Comment:
Ok I will move it and keep it as alert. My idea is to be bold enough so users get the idea quickly.